tests/test_io.py

```diff
@@ -254,3 +255,9 @@ def test_io_datapackage(test_df, tmpdir):
     # read from csv assert that IamDataFrame instances are equal
     import_df = read_datapackage(file)
     assert_iamframe_equal(test_df, import_df)
+
+
+def test_io_netcdf(test_df_year):
+    file = Path(TEST_DATA_DIR / "test_df.nc")
+    import_df = read_netcdf(file)
+    assert_iamframe_equal(test_df_year, import_df)
```

Comment:
Compare obs(erved) to exp(ected) outcome (but exp comes directly from the function).
```suggestion
    obs = read_netcdf(Path(TEST_DATA_DIR / "test_df.nc"))
    assert_iamframe_equal(obs, test_df_year)
```